src/schema/organization/__tests__/organization.spec.js

```diff
@@ -7,6 +7,70 @@ describe('Organization type tests', () => {
   const login = 'panelinhadees';
   const { query } = createTestClient(testServer);
 
+  it('organization: Organization', async () => {
+    const { data } = await(query({
+      query: queries.organizationData,
+      variables: { login },
+    }));
+    expect(data).toEqual(expectedResults.organizationData);
+  });
+
+  it('organization: { id }: Organization', async () => {
+    const { data } = await(query({
+      query: queries.organizationId,
+      variables: { login },
+    }));
+    expect(data).toEqual(expectedResults.organizationId);
```

Comment:
please use the [script](https://github.com/panelinhadees/server/blob/master/src/testUtils/integration/dataExtractor.js), as here: https://github.com/panelinhadees/server/blob/a0c39d30d45542810f6c9eda8677633a2e6fb1e9/src/schema/organization/members/__tests__/members.spec.js#L27-L30